### ISSUE TITLE
[Enhancement] Customization of Secondary ToolbarItem Icon

### ISSUE THREAD

**Issue by ske66** [xamarin/Xamarin.Forms#9181]
## Summary
Ability to change the color of the vertical dot icon attached to Secondary Toolbar item list, or change the icon completely. Currently, if you have a white/transparent toolbar the icon used to show a list of Secondary toolbar items is invisible.
## API Changes

If a user wishes to change the color of a toolbar item, they should be able to specify in xaml

```
<TabbedPage.ToolbarItems>
    <ToolbarItems.SecondaryIconColor>Green</ToolbarItems.SecondaryIconColor>
    <ToolbarItem Name="Home" Order="Secondary"/>
    <ToolbarItem Name="Print" Order="Secondary"/>
    <ToolbarItem Name="Other" Order="Secondary"/>
</ToolbarItems>
```

Or better yet, maybe restructure how these menus are created;

```
<TabbedPage.ToolbarItems>
    <ToolbarItems.Primary>
            <ToolbarItem Text="Add" Command="{Binding Add}" />
            <ToolbarItem IconImageSource="Printer.png" Command="{Binding Printer}" />
    </ToolbarItems.Primary>

    <ToolbarItems.Secondary>
        <Icon Color="Green" IconImageSource="MoreVert.png"/>
        <ToolbarItem Name="Home"/>
        <ToolbarItem Name="Print"/>
        <ToolbarItem Name="Other"/>
    </ToolbarItems.Secondary>
</ToolbarItems>
```

## Intended Use Case
This would be useful if someone is building a pure Xamarin cross-platform app using custom icons and a potentially light color Toolbar.

**Comment by MathewLC** [comment]
I was searching on how to do this when I found this issue. +1

**Comment by SmartmanApps** [comment]
Yeah, I'm just looking at these now, and I don't see any way to change text/background colour, so this would be helpful... and consistent!

**Comment by sschocke** [comment]
Was just also looking for a way to do this as our app has a white top navigation bar, and couldn't find anything.

**Comment by kcrg** [comment]
@sschocke Did you find anything? Any workaround?

**Comment by sschocke** [comment]
@kcrg Sort of... I am using the `<Shell.TitleView>` syntax on a `ContentPage` to display my own Custom navigation bar. It's obviously not optimal because you lose the standard features that Xamarin Shell gives you for free in the navigation bar... but for my particular app it turned out to be a requirement anyway as we wanted to display custom things in there.

If interested, here is some info about it:
[https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/shell/configuration#display-views-in-the-navigation-bar](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/app-fundamentals/shell/configuration#display-views-in-the-navigation-bar)

**Comment by marcojak** [comment]
This would be incredibly useful.

**Comment by Cliffy786** [comment]
Yip this would be VERY useful, I've looked everywhere and I don't like using hacks, it'd would be really convenient.
